internal/testutil/baseline/baseline.go

```diff
@@ -12,7 +14,10 @@ import (
 	"github.com/microsoft/typescript-go/internal/core"
 	"github.com/microsoft/typescript-go/internal/repo"
 	"github.com/microsoft/typescript-go/internal/tspath"
+	delta "github.com/octavore/delta/lib"
 	"github.com/pkg/diff"
+	edit "github.com/pkg/diff/edit"
+	write "github.com/pkg/diff/write"
```

Comment:
These package name aliases appear to be redundant?

(I can understand the `delta` one)

Reply:
I'm surprised format doesn't remove redundant aliases.